### PR TITLE
Allow specifying test vars for openqa-clone-custom-git-refspec

### PR DESCRIPTION
**Pull request by asdil12**
Eg:

```
openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6925 https://openqa.opensuse.org/tests/865136 INSTALLONLY=42 BETA=2
```

### CODE REVIEW

**Comment by asdil12** [comment]
@okurz

File: script/openqa-clone-custom-git-refspec [hunk 1]
```diff
@@ -39,4 +39,5 @@ build="${build:-"$repo_name#$pr"}"
 casedir="${casedir:-"$repo#$branch"}"
 GROUP="${GROUP:-0}"
 dry_run="${dry_run:-""}"
-$dry_run openqa-clone-job --skip-chained-deps --within-instance $host $job _GROUP=$GROUP TEST=$test BUILD=$build CASEDIR=$casedir PRODUCTDIR=$productdir NEEDLES_DIR=$needles_dir
+shift 2 # eat the two params so that $@ contains only test vars
+$dry_run openqa-clone-job --skip-chained-deps --within-instance $host $job _GROUP=$GROUP TEST=$test BUILD=$build CASEDIR=$casedir PRODUCTDIR=$productdir NEEDLES_DIR=$needles_dir $@
```
Comment (Martchus) [inline script/openqa-clone-custom-git-refspec]: Quote the `$@` like `"$@"` so a test variable's value can contain a white-space.

Comment (asdil12) [reply script/openqa-clone-custom-git-refspec]: done

File: script/openqa-clone-custom-git-refspec [hunk 2]
```diff
@@ -39,4 +39,5 @@ build="${build:-"$repo_name#$pr"}"
 casedir="${casedir:-"$repo#$branch"}"
 GROUP="${GROUP:-0}"
 dry_run="${dry_run:-""}"
-$dry_run openqa-clone-job --skip-chained-deps --within-instance $host $job _GROUP=$GROUP TEST=$test BUILD=$build CASEDIR=$casedir PRODUCTDIR=$productdir NEEDLES_DIR=$needles_dir
+shift 2 # eat the two params so that $@ contains only test vars
```
Comment (okurz) [inline script/openqa-clone-custom-git-refspec]: would you mind using the POSIX sh compatible `"${@:3}"` as last parameter to `openqa-clone-job` in the next line instead as in
https://github.com/okurz/scripts/blob/master/openqa-clone-set#L19

Comment (asdil12) [reply script/openqa-clone-custom-git-refspec]: Done

**Pull request merged**